docs/docs/enclave-modes.md

```diff
@@ -70,7 +70,7 @@ We test building and running release-mode enclaves on Ubuntu 20.04 LTS Server x8
     **Running** debug and release mode enclaves is not possible on macOS or Windows.
 === "Linux"
     **Running** debug and release mode enclaves requires SGX hardware and an installation of the intel
```

Comment:
Can we go through a find/replace session to replace all lowercase `intel` with an upper case? I.e. `Intel` instead of `intel`

Reply:
Done.